### ISSUE TITLE
Docs and docker image please

### ISSUE THREAD

**Issue by decole** [hwelch-fle/plankapy#1]
Hi, want test this service. How to understand what entrypoints the service has and how it works with the **planka**?

**Comment by hwelch-fle** [comment]
Right now it's hyper specific to my use case (running api calls in ArcGIS using Python).

I'm not very good with building Docker images so if you want to help out with that I'd be very grateful.

I've updated the repo to have a linked github page with pdoc auto-documentation from the docstrings in the code

**Comment by decole** [comment]
So I see https://github.com/plankanban/planka/blob/master/server/config/routes.js

and in Postman I tested the rest api, and it works without a socket)) without sockets!
I write in php. and I want to write an integration service that would work with the bar. and it turned out to be possible.
If you need to dockery, I will help you.
And thanks to your project, now I can implement a library for php enthusiasts.

**Comment by decole** [comment]
Thanks for the clarifications. Indeed getting cards from a project and a column is easier to do using a database.

**Comment by decole** [comment]
> Ah I see, this project does just implement that rest API in Python using the requests library. I think that `routes.js` file should definitely be more prominent in the planka docs.
> 
> Some of the routing on accessing items is a bit roundabout (e.g. Cards are stored in their own table, but the only way to get all the cards with the REST routes is to GET the board and iterate the cards returned in that boards included list). Most objects can be got through a unique ID, but getting that ID requires processing other requests.
> 
> Also seeing the tables that Planka uses can be helpful, and honestly finding a way to do SQL queries directly on the database would probably be an easier route to building an API than using the REST api.


But in order for connected users to instantly see changes over the web socket, you need to use Rest API.
So, if everything is done according to the canon, you need to use Rest API.

**Comment by hwelch-fle** [comment]
> But in order for connected users to instantly see changes over the web socket, you need to use Rest API.
You are correct, I've been using the database edits to make changes to thinks like task lists, and descriptions because those reload when you click on the card, but anything with moving objects between boards and lists will be best done with REST.

You can also use the database directly and create a replica for easier tracking of Planka objects in other software. I've been using the database directly to update a Metabase install with status and stage of cards.

**Comment by hwelch-fle** [comment]
Docs have now been created for the new version, Docker is coming soon
